src/components/admin/TechnicalQuestionCard.tsx

```diff
@@ -12,58 +11,24 @@ import {
   Card,
   CardHeader,
   CardBody,
-  Collapse,
   Divider,
-  Grid,
-  GridItem,
   useColorModeValue,
 } from '@chakra-ui/react';
-import {
-  ChevronDown,
-  ChevronUp,
-  Edit2,
-  Calendar,
-  Link as LinkIcon,
-  User,
-  CheckCircle,
-  Clock,
-} from 'lucide-react';
+import { ChevronDown, ChevronUp, Edit2 } from 'lucide-react';
 import type { TechnicalQuestion } from '../../types';
 
 interface TechnicalQuestionCardProps {
   question: TechnicalQuestion;
   isExpanded: boolean;
   onToggleExpand: (questionId: string) => void;
+  isAdmin?: boolean;
```

Comment:
Why are we passing this in it's in the /admin components shouldn't all of these be admin only?

Reply:
reused in interview component, can move to reg components